### PR TITLE
Fix fsOpenFileSystem

### PR DESCRIPTION
**Pull request by rajkosto**
Actually works now instead of always returning port remote dead
Thanks @misson20000 for finding out the switchbrew parameters for it were wrong

### CODE REVIEW

**Comment by rajkosto** [comment]
Maybe the 2 functions for 1.0.0 and 2.0.0+ should be merged into one, as well ?

**Comment by yellows8** [comment]
If the args are wrong update wiki.

**Comment by rajkosto** [comment]
Done

**Comment by yellows8** [comment]
"merged into one" But the args are different...

**Comment by rajkosto** [comment]
fsOpenFileSystemWithId can just ignore the titleid passed in on 1.0.0. You can often leave it at 0 or -1 on 2.0.0+ anyway.

**Comment by rajkosto** [comment]
This way is probably way more developer-friendly.

File: nx/include/switch/services/fs.h [hunk 1]
```diff
@@ -179,8 +179,8 @@ typedef enum
 } FsFileSystemType;
 
 /// Mount requested filesystem type from content file
-Result fsOpenFileSystem(FsFileSystem* out, u64 titleId, FsFileSystemType fsType); /// only on 1.0.0, only works with registered content
-Result fsOpenFileSystemWithId(FsFileSystem* out, u64 titleId, FsFileSystemType fsType, const char* contentPath); /// 2.0.0+, contentPath must be resolved manually
+Result fsOpenFileSystem(FsFileSystem* out, FsFileSystemType fsType, const char* contentPath);
```
Comment (yellows8) [inline nx/include/switch/services/fs.h]: Should add a note about sys-versions here, since either cmd can be used now.

**Pull request merged**